doc/sphinx-guides/source/installation/advanced.rst

```diff
@@ -7,22 +7,22 @@ Advanced installations are not officially supported but here we are at least doc
 .. contents:: |toctitle|
 	:local:
 
-Multiple Glassfish Servers
---------------------------
+Multiple Payara Servers
+-----------------------
```

Comment:
app server

Reply:
and all the references in this section below could be changed as well (except obviously the directory names

Reply:
Fixed in c717d63a0.